Source/Localization.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using System.Reflection;
+using System.IO;
+namespace DebugMod
+{
+    public static class Localization
+    {
+        public static Dictionary<string, string> localText;
+        public static void LoadLocaltext()
+        {
+            string path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "language.json");
```

Comment:
why like this instead of an embedded file?

Reply:
If debugmod add some new button,you can just change language.json to localize new thing ,and for different people,you can write different language.json